### PR TITLE
feat: add github pages workflow

### PR DESCRIPTION
**Pull request by anoblet**

### CODE REVIEW

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Github Pages
```
Comment (lkraav) [inline .github/workflows/github-pages.yml:1]: Is the name freeform? I feel like "Deploy Storybook" should be reflected somewhere, like both

* filename
* `name` attribute

Comment (anoblet) [reply .github/workflows/github-pages.yml]: Updated

**Review by saas786 — APPROVED** [review]
I don't see anything I can do to review it prior to merge to master?

Maybe merge first and test out and see the results?

**Comment by saas786** [comment]
@anoblet 

Seems to be working fine: https://conversionxl.github.io/aybolit/

---

chrome_62DL7nGwnR

**Comment by lkraav** [comment]
> Seems to be working fine: https://conversionxl.github.io/aybolit/

It was already manually deployed before.

https://github.com/conversionxl/aybolit/actions doesn't show this action having done anything yet.

**Comment by saas786** [comment]
> > Seems to be working fine: https://conversionxl.github.io/aybolit/
> 
> It was already manually deployed before.
> 
> https://github.com/conversionxl/aybolit/actions doesn't show this action having done anything yet.

Correct!

**Comment by anoblet** [comment]
I think I see the issue here. This should be set to `master` not `main`. I will file a PR today.

**Pull request merged**